account_invoice_refund_link/models/account_invoice.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2004-2011 Pexego Sistemas Informáticos. (http://pexego.es)
+# Copyright 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+# Copyright 2014-2018 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    refund_invoice_ids = fields.One2many(
+        "account.move", "reversed_entry_id", string="Refund Invoices", readonly=True
+    )
+
+    @api.model
+    def _reverse_move_vals(self, default_values, cancel=True):
+        move_vals = super()._reverse_move_vals(default_values, cancel)
```

Comment:
Isn't a reverse move line prepare function that you can override for not needing this heuristic to match refund lines with their originator line?

Reply:
No, there is no function 'prepare' for that, even in this method in the Odoo core, the values ​​of the lines are modified in the same way I have done, I guided by that code to do this in that way. 
https://github.com/odoo/odoo/blob/13.0/addons/account/models/account_move.py#L1870